### DISCUSSION TITLE
Unable to use regex in form fields

### DISCUSSION THREAD

**Discussion by kodecrash** [Budibase/budibase#6193] [General]
**Describe the bug**
I am trying to create multi step form and I trying to configure custom validation using regex. But it is not working.

**To Reproduce**
1. Configure Multi step form
2. Add email field
3. Add regex in the configure validate.
4. Unable to validate field even if value is correct.

![image](https://user-images.githubusercontent.com/8296994/171643651-715866ca-69ac-4e3c-aed4-0b2ddae8f7a0.png)

**Comment by aptkingston** [comment]
@kodecrash it would have been helpful if you could have provided the regex that didn't work. I've tested regex validation and it works fine - so I can only assume that the problem is the regex you are trying to use. Are you sure it is a valid expression?

I tried a basic email regex of `^[a-z]+@[a-z]+\.[a-z]+$` and it works as expected.

In future please use our discussions forum rather than opening a bug, unless you are sure that it's actually a bug. I'm going to convert this one to a discussion and if you could provide the expression you're trying to use then we can hopefully work out why it isn't working for you.

**Reply by kodecrash** [reply]
Thanks for the reply. I tried the regex with required combination and it worked.
